runtime/src/anchor/mod.rs

```diff
@@ -91,6 +105,8 @@ decl_storage! {
 decl_module! {
     pub struct Module<T: Trait> for enum Call where origin: T::Origin {
 
+        fn deposit_event<T>() = default;
```

Comment:
What is the purpose of this? Apologize in advance for my lack of knowledge on RUST/Substrate

Reply:
this is the module function that is used to send an event. `default` implies we are choosing to use the substrate's default event triggering logic